### ISSUE TITLE
Support for more marker shapes

### ISSUE THREAD

**Issue by maasha** [apexcharts/apexcharts.js#868]
Can we have additional marker shapes besides circles and squares, such as triangles, asterisks, + and x?

And for circles, squares, triangles, is it possible (perhaps already now for the circles and squares) to have open and close version (with path stroke set and optional stroke fill)?

Also, the orientation of markers, they could be rotated a number of degrees to get more different marker shapes

**Comment by pasztora** [comment]
Hi guys,
Just wanted to check in if there has been any progress on this feature-request so far? Would really love to see more markers than 2 to be able to make proper data comparison. Also it would be great if the selected markers would be visible for the legend as well (at the moment it is only a colored dot). Thanks a lot!

**Comment by maasha** [comment]
Look at the number of upvotes on this issue. The `no-issue-activity` tag added by the bot resulting in closing of this issue is a tad unfair!

**Comment by julienlaglasseadhzm** [comment]
anyone has solution to add more marker shape since 2019 ?

**Comment by flowzen1337** [comment]
can this somehow been relooked at? I would really love to see more shapes, i guess its a low hanging fruit for the devs to get more shapes in it. Would be so so nice! Or something like adding font-awesome as possible shape/icon?!

**Comment by Khalifa1997** [comment]
I agree, there should be some more available shapes

**Comment by tamerwaw2** [comment]
we need markers take more shapes, or just a custom image

**Comment by KrishnaKantwala** [comment]
Yes can someone provide if they figure out the solution for the marker shaper. We need more shapes to support inside the apex-charts

**Comment by junedchhipa** [comment]
This has been supported since **v3.51.0**. Ten shape names are accepted by `markers.shape`:

```
circle (default) | square | rect | triangle | diamond | star | sparkle | cross | plus | line
```

`rect` is an alias of `square`, so that is nine distinct shapes. It takes an array too, so each series can have its own:

```js
markers: {
  size: 8,
  shape: ['circle', 'triangle', 'diamond', 'star', 'cross'],
}
```

Per-point overrides work through `markers.discrete[].shape`, and the same set is honoured by the canvas renderer, so shapes survive `chart.renderer: 'canvas'`.

Re-tested on **6.8.0** before posting, one series per shape: all ten render, with ten distinct path geometries and the `shape` attribute set on each marker element.

The commit that added these closed #4569 and did not reference this issue, which is why this thread was never updated and then went stale. Leaving it closed, now with the release recorded. If you need a shape that is not in that list, please open a fresh issue naming it.
